modules/help_editor/php/NDB_Menu_Filter_help_editor.class.inc

```diff
@@ -118,7 +118,9 @@ class NDB_Menu_Filter_Help_Editor extends NDB_Menu_Filter
             $this->tpl_data['items'][$x][0]['value'] = $x + $count;
             $i = 1;
             foreach ($item as $key => $val) {
-                if (substr($key, "Topic") == 0) {
+                // Check if key starts with $needle ("Topic")
+                $needle = "Topic";
+                if (substr($key, 0, strlen($needle)) === $needle) {
```

Comment:
I think the logic of what this is trying to do would be a lot more intuitive if it was written as:

`if (strpos($key, "Topic") === 0)`

but since this fixes an immediate need I'll make that comment on the other, more long term PR and merge this..